packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts

```diff
@@ -61,12 +63,6 @@ export const updateProfileHandler = async ({ ctx, input }: UpdateProfileOptions)
       }
     }
   }
-  if (input.avatar) {
-    data.avatar = await resizeBase64Image(input.avatar);
-  }
-  if (input.avatar === null) {
-    data.avatar = null;
-  }
```

Comment:
It was confusing because data initially gets avatar from input as is and then we try to modify the data.

Instead of modifying we should set the correct value initially.